airflow/models/baseoperator.py

```diff
@@ -551,6 +551,15 @@ def __init__(
             )
             trigger_rule = TriggerRule.ALWAYS
 
+        if trigger_rule == "none_failed_or_skipped":
+            warnings.warn(
+                "none_failed_or_skipped Trigger Rule is deprecated. "
+                "Please use `TriggerRule.NONE_FAILED_MIN_ONE_SUCCESS`.",
```

Comment:
```suggestion
                "none_failed_or_skipped Trigger Rule is deprecated. "
                "Please use `none_failed_min_one_success`.",
```

or

```
                "TriggerRule.NONE_FAILED_OR_SKIPPED Trigger Rule is deprecated. "
                "Please use `TriggerRule.NONE_FAILED_MIN_ONE_SUCCESS`.",
```

Reply:
was copying that line https://github.com/apache/airflow/blob/35a6c302772bf5d0bb74646b2da04856c7d7aaac/airflow/models/baseoperator.py#L550  

you remark make sense